src/Microsoft.DotNet.Cli.Utils/AnsiConsole.cs

```diff
@@ -1,5 +1,5 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
Made this change as well in the E2E test, may conflict after it merges, just FYI.